### DISCUSSION TITLE
GitHub 101: How do I retrieve an exchange?

### DISCUSSION THREAD

**Discussion by kabannister** [brandon-rhodes/pyephem#218] [General]
We just exchanged messages a little while ago about computing planetary motions. How do I
retrieve that exchange? Thanks!

**Comment by brandon-rhodes** [comment]
Your GitHub Profile should show all your recent discussions — try taking a look at it, particularly you Contribution Activity:

https://github.com/kabannister

**Comment by kabannister** [comment]
OK, thanks again.

V/R,

Ken Bannister

**Reply by brandon-rhodes** [reply]
(Another GitHub usage note: if you’ll click the checkbox that marks my answer (above) as correct, then GitHub will be able to mark this Discussion as “done” so that it won’t appear as an active open question for other Skyfield users. Thanks!)

**Comment by kabannister** [comment]
I was able to retrieve our exchange of messages, but sorry - where is the
"checkbox" I should check to indicate "done"-ness?
Perhaps there is a user guide for GitHub so I don't continue wandering in
the dark?   V/R, Ken B.

**Reply by brandon-rhodes** [reply]
The checkbox is in the upper-right corner of the comment. All of GitHub's documentation is at docs.github.com; here’s the instructions for finishing out a Discussion with the checkbox:

https://docs.github.com/en/discussions/collaborating-with-your-community-using-discussions/participating-in-a-discussion#marking-a-comment-as-an-answer

**Comment by kabannister** [comment]
Got it, thanks!  - Ken Bannister
